### ISSUE TITLE
Truck 2: Add Basic Engine Analysis

### ISSUE THREAD

**Issue by Kozmocha** [UNCG-CSE/Volvo_Truck_Analytics#20]
Look at:
- EngineSpeed
- BoostPressure
- RoadInclinationVP15
- ActualEnginePercentTorque
- ActualEnginePercentTorqueHiRes

Place them all on one graph per day. All are going to need their own Axis because the value ranges are different.

**Edit: this task is dependent on #26.**

**Comment by Kozmocha** [comment]
This task is dependent on #26.

**Comment by Kozmocha** [comment]
Marking as duplicate of #19 and closing.
